ucrop/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
```

Comment:
It lack translation for two more strings - ucrop_error_input_data_is_absent and ucrop_mutate_exception_hint